### PR TITLE
Update AUTHORS

### PR DESCRIPTION
**Pull request by PedanticHacker**
Improved some comments in the AUTHORS file.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+# List of Stockfish authors
 
 # Founders of the Stockfish project and fishtest infrastructure
 Tord Romstad (romstad)
 Marco Costalba (mcostalba)
 Joona Kiiski (zamar)
 Gary Linscott (glinscott)
 
-# Authors and inventors of NNUE, training, NNUE port
+# Authors and inventors of NNUE, training, and NNUE port
```
[the remark this answers is not in the document]
Comment (PedanticHacker) [reply AUTHORS:9]: @Disservin, are you not a fan of the Oxford comma?

**Comment by PedanticHacker** [comment]
@UniQP, thank you for pointing out the mess in the alphabetical ordering of other Stockfish authors. There were many more names not in the correct place than you pointed out. I have used a tool and ordered the whole list, so now everything is precisely ordered according to the alphabet.

**Comment by PedanticHacker** [comment]
I am confident now that the AUTHORS file is ready for a merge. Please review to make sure.

File: AUTHORS [hunk 2]
```diff
@@ -200,25 +182,42 @@ Stefano Di Martino (StefanoD)
...
 Tom Vijlbrief (tomtor)
 Tomasz Sobczyk (Sopel97)
 Torsten Franz (torfranz, tfranzer)
 Torsten Hellwig (Torom)
 Tracey Emery (basepr1me)
-tttak
 Unai Corzo (unaiic)
 Uri Blass (uriblass)
 Vince Negri (cuddlestmonkey)
+borg323
```
Comment (Disservin) [inline AUTHORS:195]: how did you order this ? Now `b` is after `v` ?

Comment (peregrineshahin) [reply AUTHORS:195]: Probably ASCII code, he put all uppercase before all lowercase.

Comment (PedanticHacker) [reply AUTHORS:195]: Ah, crap! Never trust a tool, right? Well, I guess this was ordered alphabetically, but also taking capitalization into account. Will fix this, yet again. @Disservin, thanks for spotting this. I just copy and pasted the list, trusting that the tool did the right thing.

Comment (PedanticHacker) [reply AUTHORS]: I have now ordered everyone in the _other Stockfish developers_ list by hand. I think it's correct now.

**Comment by PedanticHacker** [comment]
When you have the time, please review and, if you find it valuable, merge this PR. Thank you for your helpful suggestions.

File: AUTHORS [hunk 3]
```diff
@@ -83,33 +82,33 @@ Gontran Lemaire (gonlem)
 Goodkov Vasiliy Aleksandrovich (goodkov)
 Gregor Cramer
 GuardianRM
-Günther Demetz (pb00067, pb00068)
 Guy Vreuls (gvreuls)
+Günther Demetz (pb00067, pb00068)
```
Comment (UniQP) [inline AUTHORS:86]: GNU coreutils' `sort` would revert this, because it treats ü as u.

File: AUTHORS [hunk 4]
```diff
@@ -83,33 +82,33 @@ Gontran Lemaire (gonlem)
...
+Jonathan Calovski (Mysseno)
 Jonathan McDermid (jonathanmcdermid)
 Joost VandeVondele (vondele)
-Jörg Oster (joergoster)
 Joseph Ellis (jhellis3)
 Joseph R. Prostko
 Julian Willemer (NightlyKing)
 jundery
 Justin Blanchard (UncombedCoconut)
+Jörg Oster (joergoster)
```
Comment (UniQP) [inline AUTHORS:111]: GNU coreutils' `sort` would revert this, because it treats ö as o.

File: AUTHORS (hunk 3 above)
Comment (PedanticHacker) [reply AUTHORS:86]: Well, I personally think that `ü` should come after `u`. In fact, all letter variations should come after the base letter, such as `ö` after `o`, etc.

File: AUTHORS (hunk 4 above)
Comment (PedanticHacker) [reply AUTHORS:111]: Very good eye! I made a terrible mistake here. `Jörg Oster (joergoster)` should come immediately after `Joseph R. Prostko`. Will fix this.

Comment (PedanticHacker) [reply AUTHORS]: Just pushed a new commit that fixes this. Thanks, @UniQP, for spotting this.

File: AUTHORS [hunk 5]
```diff
@@ -1,17 +1,17 @@
-# List of authors for Stockfish
+# List of Stockfish authors
 
 # Founders of the Stockfish project and fishtest infrastructure
-Tord Romstad (romstad)
-Marco Costalba (mcostalba)
 Joona Kiiski (zamar)
+Tord Romstad (romstad)
 Gary Linscott (glinscott)
+Marco Costalba (mcostalba)
```
[the remark this answers is not in the document]
Comment (PedanticHacker) [reply AUTHORS:7]: I think that it looks nicer when a listing, such as this, forms a pyramid-shaped line on the right side. The alphabetical order is not needed here, so I ordered these entries to be pyramid shaped.

File: AUTHORS [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
 
-# Authors and inventors of NNUE, training, NNUE port
+# Authors and inventors of NNUE, training, and NNUE port
 Yu Nasu (ynasu87)
-Motohiro Isozaki (yaneurao)
 Hisayori Noda (nodchip)
+Motohiro Isozaki (yaneurao)
```
[the remark this answers is not in the document]
Comment (PedanticHacker) [reply AUTHORS:12]: Well, I thought that a nice pyramid-shaped line on the right side looks better.

**Comment by PedanticHacker** [comment]
Is there anything else that needs to be done regarding the AUTHORS file? If not, please consider merging this PR with the master branch.

File: AUTHORS [hunk 7]
```diff
@@ -1,17 +1,17 @@
-# List of authors for Stockfish
+# List of Stockfish authors
 
 # Founders of the Stockfish project and fishtest infrastructure
```
Comment (vondele) [inline AUTHORS:3]: the authors here are not alphabetical.

File: AUTHORS [hunk 8]
```diff
@@ -1,17 +1,17 @@
...
-Tord Romstad (romstad)
-Marco Costalba (mcostalba)
 Joona Kiiski (zamar)
+Tord Romstad (romstad)
 Gary Linscott (glinscott)
+Marco Costalba (mcostalba)
 
-# Authors and inventors of NNUE, training, NNUE port
+# Authors and inventors of NNUE, training, and NNUE port
 Yu Nasu (ynasu87)
```
Comment (vondele) [inline AUTHORS:10]: the authors here are not alphabetical

Comment (PedanticHacker) [reply AUTHORS]: Yeah, you're right. I have now alphabetically ordered everything.

Comment (vondele) [reply AUTHORS]: no, I meant these authors are in a specific order, and there is no need to change that.

File: AUTHORS (hunk 7 above)
Comment (PedanticHacker) [reply AUTHORS]: You're right. I've reverted those. Is it good now?

**Comment by vondele** [comment]
The label just stays, one can see it is closed in a merge commit.

**Pull request closed without merging**